websockets/server.go

```diff
@@ -110,7 +110,7 @@ func (s *Server) Listen() {
 	go func() {
 		for {
 			s.Pulse()
-			time.Sleep(time.Second * 55)
+			time.Sleep(5 * time.Second)
```

Comment:
Yup. This is completely intentional. The idea is that every time the frontend receives a ping, it starts counting the time until the next one. If it receives a ping, it stops counting, and then starts again. But if ten seconds pass (ie, double the time between pings), it knows that the server has stopped pinging, which means it has died. The frontend assumes the server might come up again soon, so it tries to reconnect.

Ten seconds was chosen arbitrarily, to account for network latency. It could be anything safely above five seconds.

Does that make it clearer?